src/sentry/integrations/github/client.py

```diff
@@ -224,6 +224,25 @@ def get_commit(self, repo: str, sha: str) -> Any:
         """
         return self.get_cached(f"/repos/{repo}/commits/{sha}")
 
+    def get_merge_commit_sha_from_commit(self, repo: str, sha: str) -> str | None:
+        """
+        Get the merge commit sha from a commit sha.
+        """
+        response = self.get_pullrequest_from_commit(repo, sha)
```

Comment:
Can we add a `None` check since the function can return `Any`?

Reply:
i make sure the response is a list in the check below this line, do you mean another check specifically for `None`?

Reply:
Yeah exactly! I believe `isinstance` is fairly expensive, might help save some performance: https://stackoverflow.com/questions/42378726/why-is-checking-isinstancesomething-mapping-so-slow